docs/rules/no-duplicate-imports.md

```diff
@@ -12,59 +14,51 @@ import { find } from 'module';
 
 ## Rule Details
 
+This rules requires that all imports from a single module exists in a single `import` statement.
+
 This inspection reports any duplicated module in an import statement.
```

Comment:
Yes, that was also my impression: the second sentence repeats (from the viewpoint of problems reported) what the first sentence states (from the viewpoint of structure desired).

If the first sentence is enough (it is for the way I read the doc) then I suggest delete the second.